### ISSUE TITLE
support for editing non-numeric (character) data type

### ISSUE THREAD

**Issue by zackbatist** [MangoTheCat/dtdbshiny#1]
Hi, this is extremely helpful! However, I'm wondering how I might edit non-numeric data in a similar manner. I am not that familiar with the purrr package, but I suppose that the flatten_dbl command is to blame due to the following error that gets spit out after I try adding non-numeric characters and hitting save:
```
Can't coerce element 1 from a character to a double
```
Can you suggest a viable workaround?

**Comment by Avatoo** [comment]
Hi @zackbatist,

Yes you were right, `purrr::flatten_dbl` was not supposed to work with non-numeric and even before that I manually converted the value with `as.numeric`. Since this issue and the other one opened, I looked into the character type editing problem and made couple changes to the [app.R](https://github.com/MangoTheCat/dtdbshiny/blob/master/app/app.R) script. Now it should cope with both numeric and character types. Let me know if you have other questions.
